src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -43,9 +43,11 @@ public static MvcForm BeginForm(this IHtmlHelper htmlHelper)
         /// match the current request.
         /// </summary>
         /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
-        /// <param name="suppressAntiforgery">
-        /// If <c>true</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token. By
-        /// default &lt;form&gt; elements will automatically include an antiforgery token.
+        /// <param name="antiforgery">
+        /// If <c>true</c>, &lt;form&gt; elements will include an antiforgery token.
+        /// If <c>false</c> suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token.
```

Comment:
Also better if you use the longer description of what gets generated in the first (`If <c>true</c>`) sentence.